src/commands/MigrationRevertCommand.ts

```diff
@@ -45,7 +45,9 @@ export class MigrationRevertCommand {
         } catch (err) {
             console.error(err);
             // throw err;
+        } finally {
+            process.exit(0);
```

Comment:
its not a fix, we need to find a reason why its not finishing instead of forcely close it.